src/models/monster/index.ts

```diff
@@ -48,6 +54,46 @@ const ActionSchema = new Schema<Action>({
   actions: { type: [ActionOptionSchema], index: true },
 });
 
+const ArmorClassSchema = new Schema<ArmorClass>(
+  {
+    _id: false,
+    type: { type: String, index: true, enum: ['dex', 'natural', 'armor', 'spell', 'condition'] },
+  },
+  { discriminatorKey: 'type', _id: false }
+);
+
+ArmorClassSchema.discriminators = {};
+ArmorClassSchema.discriminators.dex = new Schema<ArmorClassDex>({
+  _id: false,
+  value: { type: Number, index: true },
+});
+
+ArmorClassSchema.discriminators.natural = new Schema<ArmorClassNatural>({
+  _id: false,
+  value: { type: Number, index: true },
+});
+
+ArmorClassSchema.discriminators.armor = new Schema<ArmorClassArmor>({
+  _id: false,
+  value: { type: Number, index: true },
+  armor: { type: [APIReferenceSchema], index: true },
+  desc: { type: String, index: true },
+});
+
+ArmorClassSchema.discriminators.spell = new Schema<ArmorClassSpell>({
+  _id: false,
+  value: { type: Number, index: true },
+  spell: { type: APIReferenceSchema, index: true },
+  desc: { type: String, index: true },
+});
+
+ArmorClassSchema.discriminators.condition = new Schema<ArmorClassCondition>({
+  _id: false,
+  value: { type: Number, index: true },
+  condition: { type: APIReferenceSchema, index: true },
+  desc: { type: String, index: true },
+});
```

Comment:
Since `value` is on all the discriminators, I think it would make more sense to include it in the original schema than in each discriminator.